### ISSUE TITLE
Support for Nextcloud Gpodder Sync

### ISSUE THREAD

**Issue by mase76** [gpodder/gpodder#1251]
Hi!
Please add support for the Nextcloud Gpodder Sync.
https://github.com/thrillfall/nextcloud-gpodder/

**Comment by auouymous** [comment]
It replicates the gpodder.net API, does it not work if you just add the nextcloud server in the gpodder.net preferences tab?

**Comment by mase76** [comment]
No, that does not work. I also tried <serveraddress>//index.php/apps/gpoddersync/ and <serveraddress>/apps/gpoddersync/. However the login should be done via app token, because some Nextcloud instances use 2FA.

**Comment by elelay** [comment]
Hi,
it doesn't replicate the gPodder api: devices are not implemented and paths for the two endpoints they reimplemented are different. It can't work as-is. The Nextcloud 2FA or Nextcloud login flow (requires a webview or opening the system's web browser) might be a red herring since it's possible to generate an app "legacy" password from user settings in Nextcloud.

**Comment by elelay** [comment]
That said an easy self-hosted version of mygpo for basic sync is really a worthy goal!

**Comment by robert-winkler** [comment]
Hi, is this still an active topic? I would love to sync gpodder with my nextcloud gpodder sync service. I can sync between antennaPOD and Kasts, but that does not help me since I want to sync an external, "stupid" mp3 player..

**Comment by robert-winkler** [comment]
I see, thanks for this acclaration @elelay. Thus, the nextcloud/antennaPOD/kasts friends should rename it...
Is there any other way to sync gPodder between devices without using gpodder.net? I'm trying to register for days, but gpodder.net seems to be broken..

**Comment by Fruchtzwerg94** [comment]
Same for me. An extension of gpodder would be awesome here.

**Comment by elelay** [comment]
There seems to be interest in syncing from different tools various servers in https://github.com/thrillfall/nextcloud-gpodder/discussions/91#

**Comment by sylph520** [comment]
yes, since gpodder.net often crashes, i think an integration with next cloud would be nice
